package.json

```diff
@@ -1,11 +1,25 @@
 {
     "devDependencies": {
+        "@types/chai": "^4.0.4",
+        "@types/chai-as-promised": "^7.1.0",
         "@types/mkdirp": "^0.5.1",
+        "@types/mocha": "^2.2.43",
+        "async-file": "^2.0.2",
+        "binascii": "0.0.1",
+        "chai": "^4.1.2",
+        "chai-as-promised": "^7.1.1",
+        "ethereumjs-testrpc": "^4.1.3",
+        "ethjs-contract": "^0.1.9",
+        "ethjs-provider-http": "^0.1.6",
+        "ethjs-query": "^0.2.9",
         "fs-readfile-promise": "^3.0.0",
         "mkdirp": "^0.5.1",
+        "mocha": "^3.5.3",
         "path": "^0.12.7",
         "recursive-readdir": "^2.2.1",
         "solc": "^0.4.16",
```

Comment:
The contracts don't currently compile with solc `0.4.16`.  We should change this to `"solc": "0.4.15"` until we can take the time to fix all of the warnings.